app/src/main/cpp/skyline/services/account/IAccountServiceForApplication.h

```diff
@@ -6,17 +6,30 @@
 #include <services/base_service.h>
 #include <services/serviceman.h>
 
-namespace skyline::service::account {
-    /**
-     * @brief IAccountServiceForApplication or acc:u0 provides functions for reading user information (https://switchbrew.org/wiki/Account_services#acc:u0)
-     */
-    class IAccountServiceForApplication : public BaseService {
-      public:
-        IAccountServiceForApplication(const DeviceState &state, ServiceManager &manager);
-
+namespace skyline {
+    namespace service::account {
         /**
-         * @brief This provides information about the running application for account services to use (https://switchbrew.org/wiki/Account_services#InitializeApplicationInfoV0)
+         * @brief This hold an accounts user ID
```

Comment:
"This structure holds an account's User ID (https://switchbrew.org/wiki/Account_services#Uid)"